src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -21,9 +21,12 @@ public class GetVerbCommand : Cmdlet
         [Parameter(ValueFromPipeline = true, ValueFromPipelineByPropertyName = true, Position = 0)]
         public string[] Verb
         {
-            get; set;
+            get => _verb;
+            set => _verb = value ?? new string[] { "*" };
         }
 
+        private string[] _verb = new string[] { "*" };
```

Comment:
@iSazonov Since code uses wildcard matching, if `-Verb` is not specified it tries to match with `null` which doesn't return any results. This would just keep it as wildcard to search by all verbs by default.

Reply:
I mean follow (before the PR) returns full verb list:
```powershell
Get-Verb -Verb $null
```